guides/dataloader/testing.md

```diff
@@ -43,7 +43,26 @@ You could also make specific assertions on the queries that are run (see the [`s
 
 ## Testing Dataloader Sources
 
-You can also test `Dataloader` behavior outside of GraphQL using {{ "GraphQL::Dataloader.with_dataloading" | api_doc }}. For example:
+You can also test `Dataloader` behavior outside of GraphQL using {{ "GraphQL::Dataloader.with_dataloading" | api_doc }}. For example, let's if you have a `Sources::ActiveRecord` source defined like so: 
+
+```ruby
+
+module Sources
+  class ActiveRecord < GraphQL::Dataloader::Source
+    def initialize(model_class)
+      @model_class = model_class
```

Comment:
I don't think this is the right `initialize` API for this example, because in the example, no `model_class` is given to `dataloader.with(...)`. Admittedly, the example below couldn't really work, unless `.id` somehow contained information about what data to fetch. But maybe instead, the example below could be clarified to be `Sources::User` instead, so that no example implementation was required.

Reply:
I was going off of this example but will rename it to Sources::User to make it more obvious.